cellmlmanip/__init__.py

```diff
@@ -1,3 +1,4 @@
 """Main module for loading, parsing and manipulating CellML models"""
 from ._config import __version__, __version_int__, version  # noqa
 from .main import load_model  # noqa
+from .model import Quantity  # noqa
```

Comment:
Is there a reason to add this in the main API?

Reply:
I added it when I was struggeling with circular imports and forgot to take it out once I had figured out how to import it properly